CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.12)
 
 project(icub_firmware_shared
-        VERSION 1.25.1)
```

Comment:
This is not a patch but rather a new feature. Hence, I would go w/ `1.26.0`.
The FW should be consistent too.

cc @marcoaccame